lms/templates/registration/password_reset_complete.html

```diff
@@ -6,49 +6,43 @@
 {% endblock %}
 
 {% block bodyextra %}
-  <script type="text/javascript">
+<script type="text/javascript">
```

Comment:
It seems that the block `bodyextra` is same for `password_reset_confirm.html`. Please make a base html and move this block there.